language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config38_negative.json

```diff
@@ -18,7 +18,7 @@
       "label": "c",
       "kind": "Variable",
       "detail": "Outer",
-      "sortText": "F",
+      "sortText": "B",
```

Comment:
Seems like the expected type here should be narrowed down. Sort text has increased which is not the expected behavior.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/39408